solr/solrj/src/test/org/apache/solr/client/solrj/impl/HttpSolrClientTestBase.java

```diff
@@ -585,17 +597,21 @@ protected void testUpdateAsync() throws Exception {
     }
   }
 
-  protected void testQueryAsync() throws Exception {
+  protected void testQueryAsync(boolean useDeprecatedApi) throws Exception {
     ResponseParser rp = new XMLResponseParser();
     DebugServlet.clear();
     DebugServlet.addResponseHeader("Content-Type", "application/xml; charset=UTF-8");
     String url = getBaseUrl() + DEBUG_SERVLET_PATH;
     HttpSolrClientBuilderBase<?, ?> b =
         builder(url, DEFAULT_CONNECTION_TIMEOUT, DEFAULT_CONNECTION_TIMEOUT).withResponseParser(rp);
     int limit = 10;
-    CountDownLatch cdl = new CountDownLatch(limit);
-    DebugAsyncListener[] listeners = new DebugAsyncListener[limit];
-    Cancellable[] cancellables = new Cancellable[limit];
+
+    CountDownLatch cdl = new CountDownLatch(limit); // Deprecated API use
```

Comment:
nitpick: just call this "latch".  Same goes elsewhere.  Abbreviations like this are hard to decipher later.

Reply:
I think I hunted these all down and changed them.